R/proportional_hazards.R

```diff
@@ -55,6 +55,29 @@
 #'   fit(Surv(time, status) ~ x, data = aml)
 NULL
 
+#' @export
+fit.proportional_hazards <- function(
+  object,
```

Comment:
Generally we leave this as `function(object,`, and the following arguments would line up with the placement of `object`, i.e. https://style.tidyverse.org/functions.html#long-lines-1